ts/Core/Renderer/SVG/Symbols.ts

```diff
@@ -402,6 +402,21 @@ function triangleDown(
     ];
 }
 
+function cross(
+    x: number,
+    y: number,
+    w: number,
+    h: number
+): SVGPath {
+    return [
+        ['M', x, y],
+        ['L', x + w, y + h],
+        ['M', x + w, y],
+        ['L', x, y + h],
+        ['z']
+    ];
+}
+
```

Comment:
Since this symbol is used only by the contour series (at least for now), I think we should keep it out of the core. See [mapmarker](https://github.com/highcharts/highcharts/blob/13f7cbf8e619bce4a88ef3df47ccb315a92f4333/ts/Series/MapPoint/MapPointSeries.ts#L222-L272) for an existing pattern to use.